src/views/Project/index.jsx

```diff
@@ -190,19 +326,23 @@ export default class Project extends Component {
         const moreGoodFirstNodes = fetchMoreResult.goodFirst.edges;
         const moreMentoredNodes = fetchMoreResult.mentored.edges;
 
-        // return previousResult;
         if (!moreGoodFirstNodes.length && !moreMentoredNodes) {
           return previousResult;
         }
 
+        pagingGood[JSON.stringify({ products, components })] =
```

Comment:
I think we can use only products here as I think it should be unique in the list.